queries.sql

```diff
@@ -11,3 +11,58 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
 UPDATE animals
 SET name = 'Agumon'
 WHERE name = 'Agumen';
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'unspecified';
+SELECT * from animals;
+ROLLBACK;
+SELECT * from animals;
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you put `SELECT COUNT(*) FROM ANIMALS;` before and after ROLLBACK here too 👍to check if your before and after results are the expected results or not 🤔